spice-carla-sync-service/Program.cs

```diff
@@ -27,8 +27,8 @@ public static void Main(string[] args)
                     logging.AddDebug();
                     logging.AddEventSourceLogger();
                 })
+                .UseUrls(urls: "http://localhost:10000")
```

Comment:
This looks like local development code, can you confirm it should be in the OpenShift builds